src/current/v25.1/deployment-operations-skills-taxonomy.md

```diff
@@ -0,0 +1,84 @@
+{{site.data.alerts.end}}
+
+## Configuration
+
+- [Verify vCPU, RAM, storage, and disk IOPS performance]({% link {{ page.version.version }}/recommended-production-settings.md %}#hardware)
+- [Configure time synchronization with NTP server]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}#step-1-synchronize-clocks)
+- [Validate network connectivity]({% link {{ page.version.version }}/known-limitations.md %}#cockroachdb-does-not-test-for-all-connection-failure-scenarios)
+
+## Security
+
+- [Create and distribute certificates; initialize cluster]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}#step-2-generate-certificates)
+- [Configure load balancer and direct a workload]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}#step-6-set-up-load-balancing)
+- [Configure RBAC]({% link {{ page.version.version }}/security-reference/authorization.md %})
+- [Encryption at rest]({% link {{ page.version.version }}/encryption.md %})
+
+## Cluster maintenance
+
+- [Shut down a node gracefully]({% link {{ page.version.version }}/node-shutdown.md %})
+- [Handling unplanned node outages]({% link {{ page.version.version }}/recommended-production-settings.md %}#load-balancing)
+- [Adding nodes]({% link {{ page.version.version }}/cockroach-start.md %}#add-a-node-to-a-cluster)
+- [Removing nodes]({% link {{ page.version.version }}/node-shutdown.md %}?filters=decommission#remove-nodes)
+- [Add a region]({% link {{ page.version.version }}/alter-database.md %}#add-regions-to-a-database)
+- [Remove a region]({% link {{ page.version.version }}/alter-database.md %}#drop-region)
+- [Rolling upgrades]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-patch-upgrade)
+- Downgrade a cluster from a [patch version]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-patch-upgrade)
+- Downgrade a cluster from a [major version]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-major-version-upgrade)
+- [Change a cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#change-a-cluster-setting)
+- Cluster repaving involves the following individual skills, which are also used during [rolling upgrades]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-patch-upgrade):
+    1. [Shut down a node gracefully]({% link {{ page.version.version }}/node-shutdown.md %})
+    1. Detach the [persistent volume]({% link {{ page.version.version }}/kubernetes-overview.md %}#kubernetes-terminology) (a.k.a. persistent disk) from the removed node's virtual machine (VM) (this step is optional but recommended)
+    1. Delete the removed node's VM
+    1. Start a new VM
+    1. Reattach the persistent disk to the new VM (necessary if you did step #2)
+    1. [Add a node to the cluster]({% link {{ page.version.version }}/cockroach-start.md %}#add-a-node-to-a-cluster) from the new VM
+
+## Troubleshooting
```

Comment:
Since this list describes problems rather than tasks, an introductory line for this section like I proposed above is definitely necessary to clarify.

Reply:
agreed, updated with an intro line saying "this is a list of common issues ..."